test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/AuthSupport.scala

```diff
@@ -35,6 +42,27 @@ trait AuthSupport {
 
   val mockAuthConnector: AuthConnector             = mock[AuthConnector]
   val mockSubscriptionService: SubscriptionService = mock[SubscriptionService]
+  def mockGetSubscriptionDetails(): Unit =
```

Comment:
the expected cgt reference and the intended response should be passed in as parameters to this function. Otherwise, you're not testing that the correct id is being passed though and you won't be able to test the error cases

Reply:
also, this looks like a duplicate of what you have put into `SubscriptionServiceSupport`

Reply:
Done.